ArchiSteamFarm/Steam/Interaction/Commands.cs

```diff
@@ -2411,6 +2415,46 @@ public sealed class Commands {
 			return responses.Count > 0 ? string.Join(Environment.NewLine, responses) : null;
 		}
 
+		private async Task<string?> ResponsePointsBalance(ulong steamID) {
+			if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
+				throw new ArgumentOutOfRangeException(nameof(steamID));
+			}
+
+			if (!Bot.HasAccess(steamID, BotConfig.EAccess.Master)) {
+				return null;
+			}
+
+			if (!Bot.IsConnectedAndLoggedOn) {
+				return FormatBotResponse(Strings.BotNotConnected);
+			}
+
+			int? points = await Bot.ArchiWebHandler.GetPointsBalance().ConfigureAwait(false);
+
+			return FormatBotResponse(points.HasValue ? string.Format(CultureInfo.CurrentCulture, Strings.BotPointsBalance, points) : Strings.WarningFailed);
+		}
+
+		private static async Task<string?> ResponsePointsBalance(ulong steamID, string botNames) {
+			if ((steamID == 0) || !new SteamID(steamID).IsIndividualAccount) {
+				throw new ArgumentOutOfRangeException(nameof(steamID));
+			}
+
+			if (string.IsNullOrEmpty(botNames)) {
+				throw new ArgumentNullException(nameof(botNames));
+			}
+
+			HashSet<Bot>? bots = Bot.GetBots(botNames);
+
+			if ((bots == null) || (bots.Count == 0)) {
+				return ASF.IsOwner(steamID) ? FormatStaticResponse(string.Format(CultureInfo.CurrentCulture, Strings.BotNotFound, botNames)) : null;
+			}
+
+			IList<string?> results = await Utilities.InParallel(bots.Select(bot => bot.Commands.ResponsePointsBalance(steamID))).ConfigureAwait(false);
+
+			List<string> responses = new(results.Where(result => !string.IsNullOrEmpty(result))!);
```

Comment:
I'd probably call `.ToList()` instead of wrapping it in a `new (...)`-call, but that's personal preference.

Reply:
If the existing ASF code he copied it from has it like that, it should be kept for consistency.

In terms of performance, there is no difference, if any, the `new` is faster.